config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
+  resources :teachers
   resources :students
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  root 'teachers#index'
```

Comment:
You probably want to move this line to the top of the file for better organization.